CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * **[Fix]** Fix Unsafe Object Deserialization.
 * **[Fix]** Fix "Collection was mutated while being enumerated" exception in MSACChannelGroupDefault.
+* **[Fix]** Disable "TREAT_WARNINGS_AS_ERRORS"
```

Comment:
It is not a fix. I would say it is improvement, but we need to clarify that we made this change to avoid blocking SDK clients if new Xcode introduces a new warning.